docs_user/modules/proc_migrating-mon-from-controller-nodes-redeploy-mon.adoc

```diff
@@ -0,0 +1,91 @@
+.Redeploy Ceph Monitor to the target node
```

Comment:
```suggestion
[id="redeploying-a-ceph-monitor-to-the-target-node_{context}"]

= Redeploying a Ceph Monitor to the target node

You use the IP address that you migrated to the target node to redeploy the Ceph Monitor on the target node.
```

Reply:
Please let me know if we should elaborate further  about what is happening in this procedure.

Reply:
I think it's good, we can eventually put notes in follow up patches in case we need to expand this section.

Reply:
Done